web/src/components/pages/profile/avatar-setup/avatar-setup.tsx

```diff
@@ -270,32 +274,60 @@ class AvatarSetup extends React.Component<Props, State> {
     refreshUser();
   }
 
+  private async deleteAvatarClip() {
+    const { api, refreshUser, addNotification, addUpload } = this.props;
+
+    try {
+      await api.deleteAvatarClip();
+      console.log('done');
+      this.setState({ clipStatus: 'notStarted' });
+      let clip = await this.props.api.fetchAvatarClip();
+      clip
+        ? this.setState({ avatarClipUrl: clip })
+        : this.setState({ avatarClipUrl: null });
+    } catch (error) {
+      console.log(error);
+    }
+
+    refreshUser();
+  }
+
   private counter = async () => {
-    this.audio.release();
-    await this.audio.init();
-    this.setState({ clipStatus: 'starting', counter: 3 });
-    const downloadTimer = setInterval(() => {
-      let tl = this.state.counter - 1;
-      this.setState({ counter: tl });
-      if (this.state.counter <= 0) {
-        this.setState({ clipStatus: 'started' });
-        const { locale } = this.props;
-        trackVoiceAvatar('create-voice-avatar', locale);
-        this.handleRecordClick();
-        clearInterval(downloadTimer);
-      }
-    }, 1000);
+    if (this.state.clipStatus === 'notStarted') {
```

Comment:
I'd recommend doing a early return here for the else case instead, i.e. `if (this.state.clipStatus !== 'notStarted') return`. That would reduce nesting which imo makes for more readable code